modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -545,7 +545,8 @@ private List<ObjectValidationRule> _getObjectValidationRules(
 			outputObjectValidationRuleSettings.add(objectValidationRuleSetting);
 		}
 
-		if (StringUtil.equals(
+		if (objectValidationRule.isActive() &&
+			StringUtil.equals(
```

Comment:
@mateussgn you are not doing a `SF` here, as you mentioned in the [commit](https://github.com/liferay-objects/liferay-portal/pull/2800/commits/95fd2876ce3f41ae35f36dea352884c9267fbf53) message. Please be careful when organizing your commits.